src/fsharp/FSharp.Core/prim-types.fs

```diff
@@ -135,7 +135,9 @@ namespace Microsoft.FSharp.Core
     type StructuralEqualityAttribute() = 
         inherit System.Attribute()
 
-    [<AttributeUsage (AttributeTargets.Class ||| AttributeTargets.Interface ||| AttributeTargets.Delegate ||| AttributeTargets.Struct ||| AttributeTargets.Enum, AllowMultiple=false)>]  
+    [<AttributeUsage (AttributeTargets.Class ||| AttributeTargets.Interface |||
+                      AttributeTargets.Delegate ||| AttributeTargets.Struct |||
```

Comment:
The `AttributeTargets.Delegate` would fit on the previous line (if we're aiming 120 symbols line width) and the whole attribute list could then be placed on two lines instead.

Reply:
When I split I often just split to approx 80 I think

Reply:
It's subjective, but I'd suggest using 120 which GitHub/Fantomas default now.